exercises/practice/palindrome-products/.articles/is-palindrome-check/content.md

```diff
@@ -0,0 +1,165 @@
+# Is number a palindrome? 
+
+There are multiple approaches to solve the palindrome product exercise. 
+Most of them require us to check if a given number is a palindrome. 
+
+## Approaches
+There are multiple ways to do it in C# and we will look at them in this article. 
+
+### String reversal
+
+Possibly the most obvious solution is to treat a number as a string of characters
+and reverse it. Let's have a look:
+
+```csharp
+using System.Linq;
+
+private static bool IsPalindrome(int number)
+{
+    var original = number.ToString();
+    var reversed = new String(original.Reverse().ToArray());
+    return original.Equals(reversed);
+}
+```
+
+If you come from some other languages you might be surprised by the above code.
+Unfortunately, C# doesn't come with a built in method to reverse a string. 
+We have to use `.Reverse()` extension method provided by 
+[Language-Integrated Query (LINQ)](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/). However, it returns an iterator and so it has to be collected with `.ToArray()` 
```

Comment:
Our [markdown guidelines](https://exercism.org/docs/building/markdown/markdown#h-links) suggest to use reference links.

Reply:
Done